guides/v2.3/install-gde/prereq/nginx.md

```diff
@@ -259,12 +261,12 @@ We recommend setting the memory limit to 2G when testing Magento. Refer to [Requ
 11. Create a new directory for the PHP session path and change the owner to the `apache` user and group:
 
 		mkdir -p /var/lib/php/session/
-		chown -R apache:apache /var/lib/php/
+		chown -R nginx:nginx /var/lib/php/
```

Comment:
@asrar7787, are you sure about this? When I originally tested these instructions a while back, I had to use the `apache` user to get this working. If you change it to the `nginx` user, I think you also have to edit your `www.conf` file in `/etc/php-fpm.d/` and restart the service:

```
user = nginx
group = nginx
```

Reply:
Related to #3238 as well.